src/frontend/src/views/Replay.vue

```diff
@@ -0,0 +1,214 @@
+<template>
+  <div class="replay">
+    <div v-bind:class="{ container: !isFullscreen, fullscreen: isFullscreen }">
+      <div class="row align-items-center">
+        <div class="col-auto mr-auto">
+          <h2 class="my-4">Replay a gamelog file</h2>
```

Comment:
I would have the name of the game. Maybe the name of the file or have some metadata of the game it self in the json.
Example: Replay game XHT4QDhjL (2019-08-14 9:14:32)

Reply:
I've used "Replay {{filename}} <small>{{roomId}}</small"